docs/workflow/blocks/http.rst

```diff
@@ -62,15 +64,32 @@ If the endpoint needs to be constructed from data, the endpoint can be specified
         }
     }
 
+.. code-block:: json
+
+    {
+    "type": "http",
+    "method": "get",
+    "endpoint": {
+        "protocol": "https:",
+        "host": "api.harvestapp.com/api/v2",
+        "pathname": "/reports/time/tasks",
+        "valueGetters": {
+            "query": "{ from: context.savedData.from, to: context.savedData.to }"
+        }
+    }
+  }
+
 
+**Headers** 
 For advanced use cases, the payload can be constructed using a JMES Path expression.
-Custom headers can also be specified using JMES Path expressions:
+Custom headers can also be specified using JMESPath expressions.
+Be careful to write your header's value with two types of quote (double quotes + single quotes), if the value is a string.
 
 .. code-block:: json
 
   {
       "type": "http",
-      "method": "post",
+      "Defaultmethod": "post",
```

Comment:
What's going on here? @gsambrotta

Reply:
Ah, thanks! I think I was typing somewhere else, and it accidentally typed on that line.